decidim/decidim
# Use local emojibase data instead of CDN

#### :tophat: What? Why?

While working in #10700, we detected that there were some requests to the jsdelivr.net domain.  

#### :pushpin: Related Issues
 
- Fixes #11069 

#### Testing

1. Sign in as user
2. Clean the emojipicker local database: in your browser (these are instructions for Firefox), open the web development tools, go to the "Storage" tab, open the "Indexed DB", delete the "PicMo-XX" databases
3. Change the language to the "Spanish" locale
4. Go to a comment and open the Emoji picker 
5. Check that the emojis are translated

#### Screenshot

![Screenshot of the translated emoji picker](https://github.com/decidim/decidim/assets/717367/64992867-b555-4ff0-9f07-1a933850201a)


:hearts: Thank you!
